htdocs/scss/pages/entry/new.scss

```diff
@@ -75,6 +75,7 @@
 
     li {
         display: inline-block;
+        margin-left: 0 !important;
```

Comment:
My understanding of CSS is that `!important` is a hack that shouldn't be used, or is there legitimate reasons to use it?